Modules/gcmodule.c

```diff
@@ -1502,27 +1502,61 @@ gc_get_referents(PyObject *self, PyObject *args)
 
 /*[clinic input]
 gc.get_objects
+    generation: Py_ssize_t(accept={int, NoneType}) = -1
```

Comment:
Can we hide `-1` from docstring?
How about ``generation: Py_ssize_t(c_default=-1) = None``?

Reply:
Done in 1294e7d

Reply:
@methane Notice that this hides the `-1` but disallows passing `None` manually:

```python
>>> gc.get_objects(generation=None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object cannot be interpreted as an integer
```

Reply:
>  How about `generation: Py_ssize_t(c_default=-1) = None`?

gc.get_objects(generation=None) must be accepted. Does it work with your proposed syntax?

Reply:
`generation: Py_ssize_t(accept={int, NoneType}, c_default="-1") = None`

Reply:
I have use
```
generation: Py_ssize_t(accept={int, NoneType}, c_default="-1") = None`
```
  in 4d8668d